omaha/client/example_test.go

```diff
@@ -66,6 +67,10 @@ func Example() {
 
 	// TODO: pass source to UpdateCheck
 	_ = source
+	if err := c.Ping(); err != nil {
+		fmt.Println(err)
+		return
+	}
```

Comment:
Was this intended to be added? 
Apart from this, everything looks fine.

Reply:
good catch ;-)